Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:alpine as node
```

Comment:
Nit: Could be nice to add a few LABELs? 

https://docs.docker.com/engine/reference/builder/#label

Version is probably not necessary yet.
Description could be good

Ex:
```
LABEL title="PantryNode"
LABEL org="ChicoState"
LABEL source="https://github.com/ChicoState/PantryNode"
```

Reply:
Skipping nit changes for now